components/Loader.tsx

```diff
@@ -1,24 +1,47 @@
 import { twMerge } from 'tailwind-merge';
 
 interface LoaderProps {
+  // eslint-disable-next-line prettier/prettier
+
+  /** The text to be displayed along with the loading animation. */
+  loaderText?: string;
+
+  /** The type of loader symbol to be displayed (Circular Animation or AsyncAPI Icon or Nothing). */
+  loaderType?: 'circular' | 'icon' | null;
```

Comment:
Instead of passing specific icon type, can we allow users to pass Icon components here?

Reply:
Done